src/MembershipsSDKAdapter.test.js

```diff
@@ -0,0 +1,58 @@
+import {isObservable} from 'rxjs';
+import {DestinationType} from '@webex/component-adapter-interfaces';
+
+import MembershipsSDKAdapter from './MembershipsSDKAdapter';
+import createMockSDK from './__mocks__/sdk';
+
+describe('Memberships SDK Adapter', () => {
+  let membershipSDKAdapter, mockSDK, meetingID;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    membershipSDKAdapter = new MembershipsSDKAdapter(mockSDK);
+    meetingID = 'meetingID';
+  });
+
+  describe('getMembersFromDestination()', () => {
+    test('returns an observable', () => {
+      expect(isObservable(membershipSDKAdapter.getMembersFromDestination())).toBeTruthy();
+    });
+
+    describe('meeting destination type', () => {
+      test('emits an array on subscription', (done) => {
+        membershipSDKAdapter.getMembersFromDestination(meetingID, DestinationType.MEETING).subscribe((members) => {
+          expect(Array.isArray(members)).toBeTruthy();
+          done();
+        });
+      });
+
+      test('throws an error on invalid meeting ID', (done) => {
+        membershipSDKAdapter.getMembersFromDestination('badID', DestinationType.MEETING).subscribe(
+          () => {},
+          (error) => {
+            expect(error).toBeDefined();
+            done();
+          }
+        );
+      });
+    });
+
+    describe('room destination type', () => {
+      test('throws an error on subscription', (done) => {
```

Comment:
Removing the nested `describe`s, we would just need to update the test description to be more specific
```suggestion
      test('throws an error on invalid meeting type', (done) => {
```

Reply:
when the room memberships functionality is added, the tests will go in this describe block.